app/forms/hub/update_ctc_client_form.rb

```diff
@@ -5,7 +5,10 @@ class UpdateCtcClientForm < ClientForm
                        :primary_first_name,
                        :primary_last_name,
                        :primary_suffix,
+                       :primary_prior_year_agi_amount,
```

Comment:
Should this form validate to either require the AGI amount or the PIN if filed 2019?

Reply:
no, because its mostly used by customer service and they can just toggle based on what they need to do without breaking the form and needing to toggle whether the person filed in 2019. Removing all of the data from the form will result in us submitting 0 values, which is the same as the client telling us they did not file.